OPERATOR.rst

```diff
@@ -666,6 +665,103 @@ one branch (see ``deployments/*.browser/environment.py``) and it will always use
 the tarball from the most recent pipeline on that branch.
 
 
+Running a ZAP vulnerability scan
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ZAP Setup
+"""""""""
+
+- Download ZAP from https://www.zaproxy.org/
+
+- Install & open ZAP
+
```

Comment:
Perhaps this section may benefit from a quick note regarding macOS requiring additional authorization to run this application not from the App Store. Additionally, there are various choices when booting ZAP if one wants the session to be persistent, is that relevant (see screenshot)?
<img width="513" alt="Screenshot 2025-05-08 at 16 50 37" src="https://github.com/user-attachments/assets/4194df49-8228-42a6-af2f-8a267f7517ad" />